### DISCUSSION TITLE
Please refrain from making nonfunctional PRs for a bit

### DISCUSSION THREAD

**Discussion by mtijanic** [NVIDIA/open-gpu-kernel-modules#190] [Announcements]
Thank you for all the enthusiasm around open-gpu-kernel-modules.

For the next while, we are going to focus review on pull requests of functional changes. Please refrain from further cosmetic pull requests until we publish our style guide.

While many of these PRs make sense in the context of this driver or general linux kernel modules, they go against the style guidelines of this codebase, or simply can't be merged into a shared codebase as they would break components that are not published (such as Windows drivers, tools like NVML, or the firmware builds).

This is on us; we should have published better guidelines on how such changes should work. Until we do, it's best to just not make any cosmetic changes to minimize the churn. I'll be tagging all such active PRs with the `cosmetic-postponed` label.

Also noted in https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/CONTRIBUTING.md

Thank you.

**Comment by afkvido** [comment]
It's kinda sad that people make PRs without reading the CONTRIBUTING.md in the first place.

**Comment by kaonkaon** [comment]
so called "programmer" on their way to "fix the typos" on big ahh code to brag themself as "contributor"
